app/controllers/users_controller.rb

```diff
@@ -267,8 +281,8 @@ def determine_follow_suggestions(current_user)
     recent_suggestions.presence || default_suggested_users
   end
 
-  def render_update_response
-    outcome = current_user.save ? "updated successfully" : "update failed"
+  def render_update_response(success)
+    outcome = success == true ? "updated successfully" : "update failed"
```

Comment:
It's relatively uncommon in Ruby to explicitly check for booleans. Most conditionals rely on truthiness (`nil`/`false` vs. everything else) instead.

Reply:
✅